base/uk.ac.stfc.isis.ibex.opis/resources/Eurotherm_single_temp.opi

```diff
@@ -0,0 +1,313 @@
+ADDR_x IOC macros are set correctly in the configuration.</text>
+    <tooltip></tooltip>
+    <transparent>true</transparent>
+    <vertical_alignment>1</vertical_alignment>
+    <visible>false</visible>
+    <widget_type>Label</widget_type>
+    <width>362</width>
+    <wrap_words>false</wrap_words>
+    <wuid>6c52603b:18311cdccfc:-6e07</wuid>
+    <x>420</x>
+    <y>-60</y>
+  </widget>
+  <widget typeId="org.csstudio.opibuilder.widgets.linkingContainer" version="1.0.0">
+    <actions hook="false" hook_all="false" />
+    <background_color>
+      <color name="ISIS_OPI_Background" red="240" green="240" blue="240" />
+    </background_color>
+    <border_color>
+      <color red="0" green="128" blue="255" />
+    </border_color>
+    <border_style>0</border_style>
+    <border_width>1</border_width>
+    <enabled>true</enabled>
+    <font>
+      <opifont.name fontName="Segoe UI" height="9" style="0" pixels="false">Default</opifont.name>
+    </font>
+    <foreground_color>
+      <color red="192" green="192" blue="192" />
+    </foreground_color>
+    <group_name></group_name>
+    <height>584</height>
+    <macros>
+      <include_parent_macros>true</include_parent_macros>
+      <EURO_NUM>$(TEMP_SENSOR)</EURO_NUM>
+    </macros>
+    <name>Linking Container</name>
+    <opi_file>Eurotherm_single.opi</opi_file>
+    <resize_behaviour>1</resize_behaviour>
+    <rules />
+    <scale_options>
+      <width_scalable>true</width_scalable>
+      <height_scalable>true</height_scalable>
+      <keep_wh_ratio>false</keep_wh_ratio>
+    </scale_options>
+    <scripts />
+    <tooltip></tooltip>
+    <visible>true</visible>
+    <widget_type>Linking Container</widget_type>
+    <width>776</width>
+    <wuid>6c52603b:18311cdccfc:-5b70</wuid>
+    <x>6</x>
+    <y>18</y>
+  </widget>
+  <widget typeId="org.csstudio.opibuilder.widgets.groupingContainer" version="1.0.0">
+    <actions hook="false" hook_all="false" />
+    <background_color>
+      <color name="ISIS_OPI_Background" red="240" green="240" blue="240" />
+    </background_color>
+    <border_color>
+      <color name="ISIS_GroupBox_Border_NEW" red="0" green="128" blue="255" />
+    </border_color>
+    <border_style>13</border_style>
+    <border_width>1</border_width>
+    <enabled>true</enabled>
+    <fc>false</fc>
+    <font>
+      <opifont.name fontName="Segoe UI" height="9" style="0" pixels="false">ISIS_GroupBox_NEW</opifont.name>
+    </font>
+    <foreground_color>
+      <color name="ISIS_OPI_Foreground" red="192" green="192" blue="192" />
+    </foreground_color>
+    <height>46</height>
+    <lock_children>false</lock_children>
+    <macros>
+      <include_parent_macros>true</include_parent_macros>
+    </macros>
+    <name>Millivolts</name>
+    <rules>
+      <rule name="EnableGroupingContainerEuro1" prop_id="visible" out_exp="false">
+        <exp bool_exp="pv0==0">
+          <value>false</value>
+        </exp>
+        <pv trig="true">$(PV_ROOT):A01:ACTIVE</pv>
```

Comment:
Similar to another comment, this is hard-coded to look at sensor 1, it should look at different sensors based on the `SENSOR` macro provided by the user.

In IBEX if you right-click in the OPI and say "dump pv list" it will show you all PVs that the OPI is trying to connect to, if you pass `SENSOR=3` then none of the PVs should be prefixed with `EUROTHRM_01:A03` for example (the only exception is the "heartbeat" pv which is special, don't worry about that one)